build_tools/python/benchmark_suites/iree/export_definitions.py

```diff
@@ -0,0 +1,27 @@
+## Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""Definitions of exported benchmark configs.
+
+The definitions should only use primitive types to be JSON serializable.
```

Comment:
We already have functionality for [de]serializing data classes. Why not use those? Otherwise I'm not sure you get much by having one level of dataclass here vs just a raw untype json object of `Dict[str, Any]`

Reply:
Yeah I think dataclasses serializing in a more native json way would be nice. Perhaps with the one addition of an `id` field